### PR TITLE
Some minor fixes for the R-2000iC/125L variant

### PR DESCRIPTION
**Pull request by gavanderhoorn**
As per subject.

Tried to push to the branch, but that didn't work.

### CODE REVIEW

File: fanuc_r2000ic_support/launch/robot_state_visualize_r2000ic125l.launch [hunk 1]
```diff
@@ -17,6 +17,12 @@
 
   <rosparam command="load" file="$(find fanuc_r2000ic_support)/config/joint_names_r2000ic125l.yaml" />
 
+  <include file="$(find fanuc_driver)/launch/robot_state.launch">
+    <arg name="robot_ip"   value="$(arg robot_ip)" />
+    <arg name="J23_factor" value="$(arg J23_factor)" />
+    <arg name="use_bswap"  value="$(arg use_bswap)" />
+  </include>
+
```
Comment (gavanderhoorn) [inline fanuc_r2000ic_support/launch/robot_state_visualize_r2000ic125l.launch:25]: Are you guys using some other driver?

Comment (simonschmeisser) [reply fanuc_r2000ic_support/launch/robot_state_visualize_r2000ic125l.launch:25]: no, but we are not using this launchfile. I still wonder where it was incorrectly copied from though

Comment (simonschmeisser) [reply fanuc_r2000ic_support/launch/robot_state_visualize_r2000ic125l.launch:25]: (and yes in a sense as we're using `fanuc_driver_exp` by you but that's robot side only ...)

Comment (simonschmeisser) [reply fanuc_r2000ic_support/launch/robot_state_visualize_r2000ic125l.launch:25]: fixed it in 165f as well

**Pull request merged**